app/views/admin/infobase/_order_form.html.haml

```diff
@@ -0,0 +1,23 @@
+%table.ui.very.basic.table
+  %thead
+    %tr
+      %th User Name
+      %th User Email
+      %th Delivery Address
+      %th Order Items
+      %th Date of Order
+      %th Price of Order
+  %tbody
+  - orders.includes(:user).each do |order|
+    %tr
+      %td #{order.user.name}
+      %td #{order.user.email}
+      %td #{order.address}
+      %td
+        - order.cart_items.includes(:menu).pluck(:title).each do |item|
+          %li
+            = item
+      %td #{order.updated_at.strftime('%Y-%m-%d %H:%M:%S')}
```

Comment:
you cannot do stuff like `.updated_at.strftime('%Y-%m-%d %H:%M:%S')` on the view. This is definitely goes to draper